iothub/service/samples/PnpServiceSamples/noComponentSample/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
+
+namespace PnpServiceSample
+{
+    public class Program
+    {        
+        // Get connection string and device id inputs
+        private static readonly string hubConnectionString = Environment.GetEnvironmentVariable("IOTHUB_CONNECTION_STRING");
+        private static readonly string deviceId = Environment.GetEnvironmentVariable("DEVICE_ID");
+
+        // These are default values as per PnP no Component device client sample.
+        private const string propertyName = "targetTemperature";
+        private const double propertyValue = 60;
+        private const string methodToInvoke = "reboot";
+        private const string methodPayload = "{\"delay\":10}";
+
+        private static ServiceClient s_serviceClient;
+        private static RegistryManager s_registryManager;
+
+        public static async Task Main(string[] args)
+        {
+            await RunSampleAsync();
+        }
+
+        private static async Task RunSampleAsync()
+        {
+            PrintLog($"Initialize the service client.");
+            InitializeServiceClient();
+
+            PrintLog($"Get Twin model Id and Update Twin");
+            await GetAndUpdateTwinAsync();
+
+            PrintLog($"Invoke a method");
+            await InvokeMethodAsync();
+
+            PrintLog($"Press any key to exit");
```

Comment:
The service client sample does not handle any callbacks, so you don't really need to have the sample hang around. You can simply exit the sample once complete.